test/TestImagePath.ts

```diff
@@ -4,6 +4,46 @@ export type TestImagePath =
   | 'align/cropped1.png'
   | 'align/croppedRef.png'
   | 'align/croppedRef1.png'
+  | 'formats/grey6.jpg'
```

Comment:
Please try to make your changes to this file without reverting the reordering that happened in https://github.com/image-js/image-js-typescript/commit/2535f4f5e3beec4733f0d8803d0be114f7f600b1 (in other words, make sure it's alphabetically sorted).